docs/src/getting-started.md

```diff
@@ -31,13 +49,66 @@ namespace ConsoleApplication
     using GraphQL.Http;
     using GraphQL.Types;
 
+    /*
+     * Database Layer
+     * In this example, it's a simple object. In the "real world,"
+     * it would likely be an object tied to an ORM, like Entity Framework.
+     */
+    public class Droid
+    {
+      public string Id { get; set; }
+      public string Name { get; set; }
+    }
+
+    /*
+     * Interface/Type Layer
+     * Every type you wish to expose via GraphQL needs to be defined.
+     * See the "Further Reading" section of the docs for ways of automatically generating schema.
+     */
+    public class DroidType : ObjectGraphType<Droid>
+    {
+      public DroidType()
+      {
+        Field(x => x.Id).Description("The Id of the Droid.");
+        Field(x => x.Name, nullable: true).Description("The name of the Droid.");
+      }
+    }
+
+    /*
+     * Query/Mutation Layer
+     * The root query is the entry point of your GraphQL service.
+     * You must have one, and only one, root query object.
+     * This object gathers together all your defined types and configures the types
+     * of searches and filters you support.
+     *
+     * In this example, the data itself is hardcoded in the `resolve:` line.
+     * In the "real world," you'd be querying your database.
```

Comment:
again this isn't really true

Reply:
How would you prefer to say it? Even if you're not querying through a direct ORM, this *is* where you fetch the actual data, however that is provisioned. The important part to me is to signal that the `resolve:` line given is *not* representative of real-world application.

Reply:
yeah "data" in the general sense is correct, im only nitpicking "database" as it connotes a specific thing. Resolve, for instance may make an http request to another REST API